js/ui/control/attribution.js

```diff
@@ -49,6 +49,15 @@ Attribution.prototype = util.inherit(Control, {
             }
         }
 
+        // remove any entries that are duplicates or strict substrings of another entry.
+        attributions.sort();
```

Comment:
Whether you sort or not, you're comparing every attribution string to every other attribution string. I don't think the sort is necessary.